content/doc/book/using/using-agents.adoc

```diff
@@ -196,3 +196,26 @@ Building remotely on agent1 in workspace /home/jenkins/workspace/First Job to Ag
 agent1
 Finished: SUCCESS
 ----
+
+=== Launching agent from console
+
+agent.jar is meant to be launched by Jenkins, typically through a remote
+shell like ssh/rsh. +
+Jenkins controller then communicates with this agent through stdin and
+stdout.
+
+agent.jar is *not* meant to initiate a connection to the controller on its
+own, so if you are trying to run +
+it from cron or as a service, you are misunderstanding how this works.
```

Comment:
Sentence per line is preferred and no need for the [line break](https://asciidoctor.org/docs/asciidoc-writers-guide/#wrapped-text-and-hard-line-breaks) that is inserted by the trailing '+'
```suggestion
The `agent.jar` is *not* meant to initiate a connection to the controller on its own.
If you are trying to run it from cron or as a service, you are misunderstanding how this works.
```